CalcualtorLogic.js

```diff
@@ -1,3 +1,4 @@
+// let currentResult;
```

Comment:
Please don't not create any variable.
you can only do changes inside the class 
Please remove this commented variable.